### DISCUSSION TITLE
局域网内其他设备无法访问alist

### DISCUSSION THREAD

**Discussion by explorer-exe-bilibili** [AlistGo/alist#4778] [Q&A]
使用香橙派的Ubuntu22.04 docker的alist，在香橙派的谷歌浏览器中用127.0.0.1:5244可以正常访问alist，但是在同一局域网下的win7笔记本用edge输入香橙派的ip加5244不能访问，提示：当前无法使用此页面192.168.1.19 未发送任何数据。
ERR_EMPTY_RESPONSE。
在笔记本上挂载alist可以正常通过127.0.0.1:5244访问，是我的访问方式有什么问题吗?alist的设置除了存储其他保持默认
其次，ping香橙派的ip也可以ping通，ssh也没问题

**Comment by explorer-exe-bilibili** [comment]
另外，香橙派的Ubuntu好像不带防火墙

**Reply by anwen-anyi** [reply]
不应该用好像 检查一下到底有没有😅

**Reply by explorer-exe-bilibili** [reply]
是指不应该用防火墙吗？我在网上找到的关闭防火墙的命令都提示有错误缺少了fairwall.server文件

**Comment by KOLDERWH** [comment]
遇到一样的问题，请问题主解决没

**Reply by explorer-exe-bilibili** [reply]
我装了宝塔面板，里面的防火墙没关，关了以后就好了

**Reply by KOLDERWH** [reply]
我是后面加上--debug参数就能运行了

**Comment by matthewlai9313** [comment]
在Windows防火墙里放行alist应用

**Comment by muxinxy** [comment]
我是把AList端口5244加入Windows防火墙的入站规则里解决局域网访问问题的

![image](https://github.com/user-attachments/assets/67292942-6494-4a10-a139-e032c086656e)
![image](https://github.com/user-attachments/assets/64ca25f1-0f89-4b91-a1d3-3d6dde2ab477)
![image](https://github.com/user-attachments/assets/646e2792-5e22-445b-999d-383cd0c7e675)
![image](https://github.com/user-attachments/assets/5ab63ab3-05f9-4b59-9196-b0089d4c114f)
![image](https://github.com/user-attachments/assets/dc99a69e-6162-4809-9eb2-a496faa464cd)
![image](https://github.com/user-attachments/assets/6170eb35-75cf-4368-beb0-9458e3d469bc)
